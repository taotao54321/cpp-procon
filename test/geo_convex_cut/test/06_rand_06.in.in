86
-998 -9
-997 -29
-980 -188
-957 -284
-942 -330
-930 -364
-914 -402
-898 -436
-858 -508
-742 -666
-714 -697
-574 -817
-540 -840
-451 -891
-435 -899
-359 -932
-278 -958
-181 -982
-105 -992
-74 -995
24 -997
126 -990
178 -982
314 -948
358 -931
440 -896
459 -886
525 -849
533 -844
573 -818
597 -799
697 -715
716 -696
806 -590
829 -557
874 -482
916 -398
931 -359
945 -322
974 -217
984 -170
987 -151
995 -84
997 -34
987 151
961 270
937 342
880 471
841 536
805 591
788 613
732 679
701 709
605 794
571 818
498 864
482 874
453 890
330 942
214 975
35 998
-37 997
-88 995
-111 992
-181 981
-212 976
-252 966
-371 927
-377 925
-395 916
-529 847
-624 778
-699 711
-729 682
-776 628
-804 592
-838 542
-859 507
-884 463
-895 440
-920 387
-937 345
-948 313
-987 148
-991 119
-996 72
50
-376 -482 -278 431
-844 223 376 264
-276 849 -960 -685
357 441 -703 -302
-692 -828 -748 -405
400 586 567 881
-224 -493 -88 -329
-891 296 611 27
357 -253 882 -641
-347 -681 540 264
-698 105 -494 638
-237 -898 -241 670
-735 -714 789 -524
950 267 639 -948
413 617 790 -930
737 -3 405 -190
802 -163 -338 986
-460 -601 622 -563
-222 -985 -50 83
-766 -520 407 -387
-554 389 -659 600
993 122 491 -983
-935 786 788 26
-737 330 -389 912
-435 -673 -850 44
982 972 -657 -831
418 65 -948 -919
374 625 -491 -676
32 -121 -913 988
-855 46 -164 -507
375 66 326 -291
-728 -958 480 -991
604 186 -527 91
404 -715 -805 -217
2 -286 -624 322
-288 -645 386 -141
-383 -539 -758 576
-515 867 -267 -892
-983 -859 -11 579
-536 685 648 -355
-537 -147 -84 935
-28 339 -199 -462
-798 981 -160 -6
776 166 -487 -502
976 619 -884 734
-440 543 63 44
-587 -753 -856 -818
-702 229 -316 -26
-22 -371 -870 -789
107 786 276 -166
