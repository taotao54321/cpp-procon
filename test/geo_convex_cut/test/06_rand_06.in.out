929357.90238035
1065944.83426698
2677372.75577126
1871469.74757463
165525.27163309
1679215.51211947
1843460.71622132
1294389.66317574
1544194.77817072
1982433.87459305
305980.71897577
1088422.59480420
2708480.51637630
99697.08529750
527806.56626154
852073.28114809
2435314.94539688
2656026.20392571
1436490.80534869
2392226.43995714
929126.96024743
100182.59597311
893033.49688677
157436.22872104
270998.79878742
1417064.72155743
1182110.24710991
1631087.55213200
1453470.57137965
2512358.81638291
855109.55075569
3117624.47339158
1829169.54650808
595290.64981792
1157620.84770083
2232959.76672740
548168.08866219
2317117.86546697
909158.70871241
1241941.79934787
715711.94269407
1757087.58362481
1835240.15104547
1134312.62120234
2800527.36967330
1410906.51000617
451870.20492870
1949824.89753844
929060.14688798
1082335.81453591
